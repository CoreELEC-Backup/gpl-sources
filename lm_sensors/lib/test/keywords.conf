# chip keyword on a line by itself
chip

# preceded by whitespace
	 chip

# with trailing whitespace
chip	 

# ditto for the rest of the keywords
label

	  label

label	 	

set

 set

set 

compute

	compute

compute	

bus

	  	bus

bus	  	

ignore

	ignore

ignore	

# keyword followed by EOL/EOF
chip
