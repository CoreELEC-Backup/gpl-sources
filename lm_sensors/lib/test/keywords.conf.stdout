2: CHIP
3: EOL
5: CHIP
6: EOL
8: CHIP
9: EOL
11: LABEL
12: EOL
13: LABEL
14: EOL
15: LABEL
16: EOL
17: SET
18: EOL
19: SET
20: EOL
21: SET
22: EOL
23: COMPUTE
24: EOL
25: COMPUTE
26: EOL
27: COMPUTE
28: EOL
29: BUS
30: EOL
31: BUS
32: EOL
33: BUS
34: EOL
35: IGNORE
36: EOL
37: IGNORE
38: EOL
39: IGNORE
40: EOL
42: CHIP
43: EOL
43: EOF
